server/ice_servers/coturn.py

```diff
@@ -0,0 +1,45 @@
+"""
+Coturn static auth api
+
+Creates ice_server record with temporary credentials computed from coturn static-auth-secret
+
+http://stackoverflow.com/a/35767224
+"""
+
+import time
+import hmac
+import base64
+from hashlib import sha1
+from server.config import COTURN_HOSTS, COTURN_KEYS, TWILIO_TTL
+
+class CoturnHMAC:
+    def __init__(self, coturn_hosts=None, coturn_keys=None):
+        self.coturn_hosts = coturn_hosts or COTURN_HOSTS
+        self.coturn_keys = coturn_keys or COTURN_KEYS
+
+    def fetch_token(self, username='faf-user', ttl=None):
+        servers = []
+
+        ttl = ttl or TWILIO_TTL
+        if ttl is not None:
+            ttl = int(ttl)
+        else:
+            ttl = 3600*24
```

Comment:
What? Why is the default hardcoded in here? Especially after a block of code which ensures that this branch will NEVER happen!

Reply:
it doesn't ensure this will never happen, TWILIO_TTL can be none.

Reply:
I see. Well it actually can't be `None` but it could be empty string, since it's defined as:
```python
TWILIO_TTL = os.getenv("TWILIO_TTL", 86400)
```
(again note the mysterious default constant, only it's written differently this time)

I don't think that we need to do error handling for the environment variables being set up wrong. And even if we do, that error handling should be done in `config.py` not in `coturn`. This file should assume that TWILIO_TTL has been set to a default correctly.